packages/client/src/runtime/core/engines/common/utils/util.ts

```diff
@@ -0,0 +1,28 @@
+import crypto from 'crypto'
+import newGitHubIssueUrl from 'new-github-issue-url'
+
+export function getGitHubIssueUrl({
```

Comment:
Most of the helpers from here have moved to `internals`, I guess that's why github thinks it is a new file.